contrib/kafka/cmd/receive_adapter/main.go

```diff
@@ -95,7 +95,7 @@ func main() {
 	stopCh := signals.SetupSignalHandler()
 
 	logger.Info("Starting Apache Kafka Receive Adapter...",
-		zap.String("bootstrapServer", adapter.BootstrapServers),
+		zap.String("bootstrap_server", adapter.BootstrapServers),
```

Comment:
Underlying yaml parameter name is `bootstrapServers` (see it is plural with "s" at the end) I think it would be good to keep it consistent for people that will search in logs when troubleshooting?
https://github.com/knative/eventing-sources/blob/master/contrib/kafka/config/300-kafkasource.yaml#L46

Reply:
This was already discussed here: https://github.com/knative/eventing-sources/pull/329#discussion_r271031783